conans/client/loader.py

```diff
@@ -1,4 +1,4 @@
-import imp
+from importlib import invalidate_caches, util
```

Comment:
`util` import is a generic name, please, rename it as `imp_util` or another more specific one

Reply:
done